test/integration/addons_test.go

```diff
@@ -796,6 +797,19 @@ func validateHeadlampAddon(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
+func validateInspektorGadgetAddon(ctx context.Context, t *testing.T, profile string) {
+	defer PostMortemLogs(t, profile)
+
+	rr, err := Run(t, exec.CommandContext(ctx, Target(), "addons", "enable", "inspektor-gadget", "-p", profile, "--alsologtostderr", "-v=1"))
+	if err != nil {
+		t.Fatalf("failed to enable inspektor-gadget addon: args: %q: %v", rr.Command(), err)
+	}
```

Comment:
Instead of doing this the convention is to add `--addons=inspektor-gadget` on line 81 and then this block of enabling the addon can be removed.